### ISSUE TITLE
Split view not working

### ISSUE THREAD

**Issue by LarryEitel** [gitobsidiantutorial/obsidian-tabs#12]
I am running Insider Build v0.12.1.

No matter what I try, I can not get side-by-side view. When I split vertically, it only adds a new tab. If I drag a tab to either side of current opened tab, it shows where the dragged tab SHOULD go, but only reverts back without showing side-by-side.

Otherwise, this is an excellent plugin!!!!!!!!!

**Comment by element4l** [comment]
Having the same problem!

**Comment by gitobsidiantutorial** [comment]
https://github.com/gitobsidiantutorial/obsidian-tabs#using-split-view

Split view has a set order of operations: it's a **horizontal** split you first need to create, in a workspace with only one pane open. Then you vertically split a pane to open tabs.

This is working in a test vault on release, but I cannot say with certainty if this works on the insider build.

**Comment by element4l** [comment]
I'm on the regular build of obsidian. When this extension is enabled and I open a horizontal split view, it instead opens a vertical split view and doesn't open tabs. If I then open a vertical split in one of the panels, it opens the tab view at the top, after which I can open new notes in tabs on that side panel. I'm not sure this is the intended behavior but either way it is very convoluted. Looking forward to future versions.

**Comment by gitobsidiantutorial** [comment]
Yes, this is intentional behaviour. I haven't managed to introduce a new type of split. Instead, the existing split types are targeted with CSS.

You can also toggle the plugin on and off to adjust the split view sizing. I recommend saving a workspace that is configured as you like, in unison with the 'starred files' plugin to quickly open files you regularly open.

https://user-images.githubusercontent.com/81381984/116575289-cb470e80-a90e-11eb-8914-18d2b2940ecf.mp4
